tests/providers/google/cloud/transfers/test_bigquery_to_sql.py

```diff
@@ -0,0 +1,29 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+from __future__ import annotations
+
+import pytest
+
+
+@pytest.mark.skip(
+    reason="FIXME: this test exists to avoid Non-DB tests from failing due to a mistyped "
+    "path in test_project_structure. Remove this file in the next Airflow's version "
+    "after 2.9.1."
+)
+class TestBigqueryToSql:
+    pass
```

Comment:
No idea what you try to achieve here, and what resolved in Airflow 2.9.1?
 
There is no tests for the module [`airflow/providers/google/cloud/transfers/bigquery_to_sql.py`](https://github.com/apache/airflow/tree/main/airflow/providers/google/cloud/transfers/bigquery_to_sql.py) that why it listed in `tests/always/test_project_structure.py::TestProjectStructure::test_providers_modules_should_have_tests`. We expect to have this test eventually: https://github.com/apache/airflow/issues/35442

So better return back `tests/providers/google/cloud/transfers/test_bigquery_to_sql.py` in `tests/always/test_project_structure.py::TestProjectStructure::test_providers_modules_should_have_tests` rather than placeholder test.

Reply:
My bad, I missed this file in the `src` :)
I thought that it has been removed or something

Reply:
Some useful command for found previously deleted file in local git history

```console
# Last 5 changes (`-5` flag) in specified file
❯ git log --all -5 -- tests/providers/google/cloud/transfers/test_bigquery_to_sql.py
```